src/interface/hooks/use-get-looped-present-proof-records.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * This function returns continuously the present proof records
+ */
+import { useCallback, useState } from 'react'
+import getLooped from '../api/helpers/get-looped'
+export default function useGetLoopedPresentProofRecords() {
+  const path = '/present-proof-2.0/records'
+  const transformData = (retData) => retData.results
+  const statusOptions = ['started', 'error', 'stopped']
+  const [status, setStatus] = useState(statusOptions[0])
+  const [error, setError] = useState(null)
+  const onStartFetch = useCallback((origin, state = null, setStoreData) => {
+    const params = state ? `state=${state}` : {}
+    const intervalId = setInterval(() => {
+      getLooped(
+        origin,
+        path,
+        params,
+        setStatus,
+        setError,
+        setStoreData,
+        transformData
+      )
+    }, 3000)
```

Comment:
That looks like a magic number to me. Why 3s?

Reply:
I'll set it back to 1s